JASP-Engine/JASP/R/correlation.R

```diff
@@ -513,7 +513,7 @@ Correlation <- function(jaspResults, dataset, options){
   shapiroTable$showSpecifiedColumnsOnly <- TRUE
   
   shapiroTable$addColumnInfo(name = "var1",      title = "",                      type = "string")
-  shapiroTable$addColumnInfo(name = "separator", title = "",                      type = "string")
+  shapiroTable$addColumnInfo(name = "separator", title = "",                      type = "separator")
```

Comment:
Hmm, would it maybe be nice to add a function to jaspTable called addColumnSeparator or something? It can simply do this and be in zzzWrappers for instance.

Reply:
Yes, standardizing this would probably be a good idea!